daisy/step_deprecate_images.go

```diff
@@ -76,10 +83,16 @@ func (d *DeprecateImages) run(ctx context.Context, s *Step) DError {
 		wg.Add(1)
 		go func(di *DeprecateImage) {
 			defer wg.Done()
-
-			w.LogStepInfo(s.name, "DeprecateImages", "%q --> %q.", di.Image, di.DeprecationStatus.State)
-			if err := w.ComputeClient.DeprecateImage(di.Project, di.Image, &di.DeprecationStatus); err != nil {
-				e <- newErr("failed to deprecate images", err)
+			if di.DeprecationStatusAlpha.State != "" {
+				w.LogStepInfo(s.name, "DeprecateImages", "%q --> %q with DefaultRolloutTime %s.", di.Image, di.DeprecationStatusAlpha.State, di.DeprecationStatusAlpha.StateOverride.DefaultRolloutTime)
+				if err := w.ComputeClient.DeprecateImageAlpha(di.Project, di.Image, &di.DeprecationStatusAlpha); err != nil {
+					e <- newErr("failed to deprecate images", err)
+				}
+			} else {
+				w.LogStepInfo(s.name, "DeprecateImages", "%q --> %q.", di.Image, di.DeprecationStatus.State)
+				if err := w.ComputeClient.DeprecateImage(di.Project, di.Image, &di.DeprecationStatus); err != nil {
+					e <- newErr("failed to deprecate images", err)
+				}
```

Comment:
Minor: writing to `e` channel can be consolidated in one place outside the if/else

Reply:
Made this change in [90f4a2d](https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1585/commits/90f4a2d7d536a8a77324428d4ba9a2a0757eaf8f)